src/antelope/stores/chain.ts

```diff
@@ -0,0 +1,154 @@
+/**
+ * Chain: This store is responsible for managing specific data for each registered
+ * chain to facilitate access and reading of its data.
+ *
+ * It handles both native Antelope chains and EVM chains. For each chain, a Chain object
+ * is defined containing the specific data for the chain, such as name, endpoint,
+ * token, etc.
+ *
+ * Internally, two chains are maintained, identified as the chain where the logged-in user
+ * is located (loggedChain) and the chain being explored by the user (currentChain), which
+ * may or may not coincide. The application can change the current chain at any time,
+ * providing access to the data and accounts for that chain. On the other hand, the logged
+ * chain is where the logged-in user is located and is where transactions are executed.
+ * This last chain is automatically changed when the user logs in or logs out.
+ */
+
+
+import { defineStore } from 'pinia';
+import {
+    createInitFunction,
+    createTraceFunction,
+    useFeedbackStore,
+} from 'src/antelope/stores/feedback';
+import { ChainSettings } from 'src/types';
+
+// main native chains
+import EOS from 'src/antelope/chains//native/eos';
+import Telos from 'src/antelope/chains//native/telos';
+import UX from 'src/antelope/chains//native/ux';
+import Wax from 'src/antelope/chains//native/wax';
+
+// test native chains
+import TelosTestnet from 'src/antelope/chains//native/telos-testnet';
+import Jungle from 'src/antelope/chains//native/jungle';
+
+// main evm chains
+import TelosEVM from 'src/antelope/chains//evm/telos-evm';
+
+// test evm chains
+import TelosTestnetEVM from 'src/antelope/chains//evm/telos-testnet-evm';
```

Comment:
should these imports have the double slash? e.g. `'src/antelope/chains//native/ux'` between 'chains' and 'native'